sdk/core/core-lro/review/core-lro.api.md

```diff
@@ -10,11 +10,26 @@ import { AbortSignalLike } from '@azure/abort-controller';
 export type CancelOnProgress = () => void;
 
 // @public
-export interface LongRunningOperation<T> {
-    requestMethod: string;
-    requestPath: string;
-    sendInitialRequest: () => Promise<LroResponse<T>>;
-    sendPollRequest: (path: string) => Promise<LroResponse<T>>;
+export function createPoller<TResult, TState extends OperationState<TResult>>(lro: LongRunningOperation, options?: CreatePollerOptions<TResult, TState>): Promise<SimplePollerLike<TState, TResult>>;
+
+// @public
+export interface CreatePollerOptions<TResult, TState> {
+    intervalInMs?: number;
+    processResult?: (result: unknown, state: TState) => TResult;
+    resourceLocationConfig?: LroResourceLocationConfig;
+    restoreFrom?: string;
```

Comment:
Is serialization/restoration so generalized that we can implement it abstractly in the core poller? I worry that this will have the same problems that `resumeFrom` has now.

Reply:
Could you elaborate on what are the problems with poller state serialization/deserialization?

Reply:
This seems to be being discussed in another thread so I will mark this as resolved for now.